### DISCUSSION TITLE
url for NEST camera?

### DISCUSSION THREAD

**Discussion by lordquazi** [Sunoo/homebridge-camera-ffmpeg#1264] [Q&A]
trying to add this public camera with no luck.  
https://video.nest.com/embedded/live/fq05SnaOmX/live.sdp

any ideas?  thanks.

**Comment by Sunoo** [comment]
As far as I know, Nest’s setup doesn’t work with this plugin directly (I have no Nest products myself). For Nest stuff, I believe the current plugin is still homebridge-nest-cam.

**Comment by lordquazi** [comment]
thank you — didn’t even think of that!

I’ll see if I can add this public cam to it.

Take care,
